Tabloid/client/src/components/CategoryList.js

```diff
@@ -17,15 +18,20 @@ const CategoryList = () => {
   return (
     <div>
       <div className="container">
-        <div className="row justify-content-center">
+      <Link to="/categories/add" className="category-link">
+        Add A New Category
+      </Link>
+      </div>
+      <div className="container">
+        <div className=" mt-4 row justify-content-center">
           {categories.map((category) => (
-            <Category category={category} key={category.id} />
+            <ListGroupItem><Category category={category} key={category.id} />
+            <Link to={`/categories/edit/${category.id}`}>Edit</Link>{" "}
```

Comment:
I'm getting a key warning, might need to add keys to this <Link> component

Reply:
Fixed!